altiProfilAdmin/locales/fr_FR/admin.UTF-8.properties

```
@@ -19,7 +19,7 @@ form.srid.label=SRID
 form.ignServiceKey.label=Clé IGN
 form.ignServiceUrl.label=URL du service IGN
 form.altiresolution.label=Résolution
-form.connection_check.label=Table requêtable ?
+form.connection_check.label=Status de la connection
```

```suggestion
form.connection_check.label=Status de la connexion
```
https://www.lalanguefrancaise.com/orthographe/connexion-ou-connection-orthographe ;-)

> On écrit toujours « connexion » en français : si en anglais on écrit « connection » avec un « t », en français, l'orthographe de ce mot est toujours « connexion » avec un « x ». Malgré le fait que le verbe associé est « connecter », connexion s'écrit toujours avec un « x ».